nanome/_internal/logs.py

```diff
@@ -199,8 +232,9 @@ def create_console_handler():
         handler.setFormatter(color_formatter)
         return handler
 
-    @staticmethod
-    def log_record(record):
-        """When a log record is received through the PipeHandler, log it."""
-        record_logger = logging.getLogger(record.name)
-        record_logger.handle(record)
+    def poll_for_logs(self):
+        has_data = self.main_pipe_conn.poll()
```

Comment:
If I understand the code well, that change is going to do weird stuff with the process API, since you removed the _DataType.log check, it won't know if the message is coming for the process API or for the log manager

Reply:
Hmmmm, okay, I think you created a second Pipe, right?

Reply:
Yep, @ajm13 pointed out we kind of hacked the log data into the process pipe. With fresh eyes it was pretty easy to make it it's own pipe

Reply:
I wouldn't say it was hacked, since the goal was to avoid having unnecessary pipes, but it makes the intent clearer that way and we don't have enough subprocesses to get to the point where having too many sockets becomes a problem, so that works!
However you can probably remove the _DataType.process now, since it's not useful anymore

Reply:
I tried removing the whole _Datatype class, but it's a more integral than I realized. I removed the check in _sessions, so that's as good as I could get it
https://github.com/nanome-ai/nanome-lib/pull/275/commits/111e8b0021f905e9c2465ca5b66510f86ff79552

Reply:
It looks like there are two _DataType classes 🤔 Alright, fine with me!

Reply:
Tried again, and I see a way forward, but it should definitely be its own PR